winform/Forms/MainForm.Designer.cs

```diff
@@ -717,32 +720,33 @@ private void InitializeComponent()
             this.tabComfort.Text = "Comfort/Cosmetics";
             this.tabComfort.UseVisualStyleBackColor = true;
             // 
-            // lGossip
-            // 
-            this.lGossip.AutoSize = true;
-            this.lGossip.BackColor = System.Drawing.Color.Transparent;
-            this.lGossip.Font = new System.Drawing.Font("Microsoft Sans Serif", 8.25F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
-            this.lGossip.ForeColor = System.Drawing.Color.Black;
-            this.lGossip.Location = new System.Drawing.Point(130, 112);
-            this.lGossip.Name = "lGossip";
-            this.lGossip.Size = new System.Drawing.Size(98, 13);
-            this.lGossip.TabIndex = 20;
-            this.lGossip.Text = "Gossip Stone hints:";
-            // 
             // cGossipHints
             // 
             this.cGossipHints.DropDownStyle = System.Windows.Forms.ComboBoxStyle.DropDownList;
             this.cGossipHints.FormattingEnabled = true;
             this.cGossipHints.Items.AddRange(new object[] {
             "Default",
+            "Dark",
```

Comment:
This isn't right. These aren't gossip stone related. This is tatl color.

Reply:
Technically fine as is because the new databinding system re-populates the list. Fixed this by deleting the items